src/rtctools/optimization/pi_mixin.py

```diff
@@ -278,6 +271,13 @@ def timeseries_export(self):
         """
         return self.__timeseries_export
 
+    @property
+    def data_config(self):
```

Comment:
This addition should not be part of this PR

Reply:
We do need this when reading a seed from a given xml file. It is also used in the MultiSeedMixin test. I will move it to a separate commit and perhaps make a separate PR out of it.

Reply:
I've moved it to a separate commit.